aspnetcore/blazor/components.md

```diff
@@ -22,14 +22,6 @@ Components are implemented in [Razor](xref:mvc/views/razor) component files (*.r
 
 A component's name must start with an uppercase character. For example, *MyCoolComponent.razor* is valid, and *myCoolComponent.razor* is invalid.
 
-Components can be authored using the *.cshtml* file extension as long as the files are identified as Razor component files using the `_RazorComponentInclude` MSBuild property. For example, an app that specifies that all *.cshtml* files under the *Pages* folder should be treated as Razor components files:
-
-```xml
-<PropertyGroup>
-  <_RazorComponentInclude>Pages\**\*.cshtml</_RazorComponentInclude>
-</PropertyGroup>
```

Comment:
This is an old workaround. We don't need to tell people about this anymore, I can't think of a case where we'd recommend it, and we might remove this in the future.